### ISSUE TITLE
Application in nifti images

### ISSUE THREAD

**Issue by Nilser3** [cornerstonejs/calculate-suv#23]
Hi @jbocce @salimkanoun 

I have seen this tool with great joy,

I would like to be able to make SUV measurements in certain regions of the brain in PET CT and SPECT CT,
(Specifically, I would like to calculate SUVs in brain regions segmented by deep learning algorithms.)

However, I still don't know if it can work with nifti images.
Likewise, it is not clear to me how to use the tool?

Do I need JavaScript?

Thank you!

best,

Nilser

**Comment by salimkanoun** [comment]
Hi, i don't think you will be able to use it in Nifti file because the SUV calculation needs a lot of dicom metadata (injected dose, injected time, acquisition type, manufacturer...)
All these information are usally not available in Nifti files.
So the SUV computation has to be done from dicom image and then writen in Nifti using calculated SUV values

**Comment by Nilser3** [comment]
thanks for the reply @salimkanoun ,

I understand !

So how could I use this tool with DICOM data?


Best,

**Comment by salimkanoun** [comment]
yes, I don't remember the code exactly it might be specific to cornerstone metadata representation however the code should be easy to extract to make your own implementation in any programming language you want
